src/ios/PushPlugin.m

```diff
@@ -21,9 +21,17 @@
  LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE
  OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF
  ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ 
+ 
+ 
+ ADAPTED for experteer, example push
+ 
+ {"aps":{"alert":"Hello from APNs Tester.","badge":1,"sound":"default","category" : "actions"},"p":"/career/profile"}
+ 
  */
 
 #import "PushPlugin.h"
+#import "Constants.h"
```

Comment:
Hi Taras,

you are right. Sorry, I did forget to commit these files. They belong to the modified PushPlugin. We needed the PushPlugin to be modified for the interactive notifications. 
Now everything shote be commited..

ok?

All the best,
Sascha